clang/include/clang/Analysis/Analyses/LifetimeSafety/LifetimeSafety.h

```diff
@@ -42,6 +42,11 @@ class LifetimeSafetyReporter {
   virtual void reportUseAfterFree(const Expr *IssueExpr, const Expr *UseExpr,
                                   SourceLocation FreeLoc,
                                   Confidence Confidence) {}
+
+  virtual void reportUseAfterReturn(const Expr *IssueExpr,
+                                    const Stmt *EscapeStmt,
```

Comment:
Does it have to be a `stmt` ?
I would be more specific here `const Expr *EscapedExpr,`